src/sequelize.d.ts

```diff
@@ -129,9 +129,9 @@ export interface ConnectionOptions {
  * Interface for replication Options in the sequelize constructor
  */
 export interface ReplicationOptions {
-  read: ConnectionOptions[];
+  read: ConnectionOptions[] | string[];
```

Comment:
Users are allowed to use a mix of both formats :)

```suggestion
  read: Array<ConnectionOptions | string>;
```